packages/angular/cli/commands/doc-impl.ts

```diff
@@ -41,13 +41,13 @@ export class DocCommand extends Command<DocCommandSchema> {
         /* tslint:disable-next-line:no-implicit-dependencies */
         const currentNgVersion = require('@angular/core').VERSION.major;
         domain = `v${currentNgVersion}.angular.io`;
-      } catch (e) {}
+      } catch (e) { }
     }
 
     let searchUrl = `https://${domain}/api?query=${options.keyword}`;
 
     if (options.search) {
-      searchUrl = `https://www.google.com/search?q=site%3A${domain}+${options.keyword}`;
+      searchUrl = `https://${domain}/?search=${options.keyword}`;
```

Comment:
I just realized that for archive version (i.e. older than the current stable), non-docs pages (including the home page) are redirected to `/docs` (losing the query params). So, this URL should be `https://${domain}/docs?search=${options.keyword}` 😅
(My bad for suggesting the wrong URL 😇)

I submitted #16856.